cluster/log-dump/log-dump.sh

```diff
@@ -482,14 +483,13 @@ function dump_nodes_with_logexporter() {
       proc=${max_dump_processes}
       wait
     fi
-  done
+  done < <("${KUBECTL}" get pods -n "${logexporter_namespace}" -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.nodeName}{"\n"}{end}')
   # Wait for any remaining processes.
   if [[ proc -gt 0 && proc -lt ${max_dump_processes} ]]; then
     wait
   fi
 
   # List registry of marker files (of nodes whose logexporter succeeded) from GCS.
-  local nodes_succeeded
```

Comment:
I remove `nodes_succeeded` because shellcheck detects the below and I cannot find it in anywhere.
```
In ./cluster/log-dump/log-dump.sh line 492:
  local nodes_succeeded
        ^-------------^ SC2034: nodes_succeeded appears unused. Verify use (or export if used externally).
```